### ISSUE TITLE
Temple of Apshai Trilogy (1982)

### ISSUE THREAD

**Issue by TimberlAndRE-de** [exoscoriae/eXoDOS#720]
wrong version  it's for the atari/commodore game version

**Comment by SmilingSpectre** [comment]
Wrong. It's universal manual again. It needs specific PC loading instructions though, that are not found. (Is it really worth it?)

**Comment by TimberlAndRE-de** [comment]
@SmilingSpectre you're right, the manual says: _"IBM or Macintosh Apshai Guide: If you purchased an IBM PC or Apple Macintosh version, a special "handbook" has been included that will contain important operating system and game play instructions"_

So i think it's worth to find as for preservation and information reasons
